testnets/sourcetestnet/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "sourcetestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Source Testnet",
+  "chain_id": "source-testnet",
+  "bech32_prefix": "source",
```

Comment:
Please register this bech32 prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done.

https://github.com/satoshilabs/slips/commit/2d57faf26bb97b8e9cb808dbbfb9c8571fb7c85f